activitystreams.py

```diff
@@ -88,6 +89,11 @@ def get(self):
     elif site == 'facebook':
       src = facebook.Facebook(
         access_token=util.get_required_param(self, 'access_token'))
+    elif site == 'flickr':
+      src = flickr.Flickr(
+        access_token_key=util.get_required_param(self, 'access_token_key'),
+        access_token_secret=util.get_required_param(self, 'access_token_secret'),
+        user_id=util.get_required_param(self, 'user_id'))
```

Comment:
is this required here? i get that we need it for `get_actor()`, but i think we only need the actor for some output types, eg. atom, right?